### ISSUE TITLE
testbed.html functionality improvement

### ISSUE THREAD

**Issue by mohsinhaider** [AmadeusITGroup/Accessibility.js#1]
testbed.html, a core component of Amadeus' flagship accessibility open source project, Accessibility.js, has a great feature set and provides a functionality that is very important to this project's mission. However, there are features missing from this project. I am not volunteering to develop this feature set, nor am I suggesting that the current implementation is lacking, but instead that it has the opportunity to change the way web developers think about accessibility.

Current Functionality:
```
<!DOCTYPE html>
<html>
<head>
</head>
<body>
</body>
```
</html>`

Proposed Functionality:
```
<!DOCTYPE html>
<html>
<head>
<!-- Page title here -->
</head>
<body>
<!-- Main functionality -->
</body>
</html>
```
